shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -113,6 +113,12 @@ class FlutterPlatformViewsController {
   std::map<std::string, fml::scoped_nsobject<NSObject<FlutterPlatformViewFactory>>> factories_;
   std::map<int64_t, fml::scoped_nsobject<NSObject<FlutterPlatformView>>> views_;
   std::map<int64_t, fml::scoped_nsobject<FlutterTouchInterceptingView>> touch_interceptors_;
+  // Mapping a platform view ID to the index of the root_view under `flutter_view_.subviews`
+  //
+  // 0 index means represents the first item in the `flutter_view_.subviews`.
+  // See also:
+  // * `root_views_`
+  std::map<int64_t, int64_t> root_views_indices_;
```

Comment:
nit: maybe view_positions_ (skimming on a high level here I think it's easier to think of this as the position of the platform view amongst the children of flutter_view more details about the view chains are in the comment)